lib/ui/screens/contact_list_screen.dart

```diff
@@ -11,7 +11,7 @@ class ContactList extends StatefulWidget {
 
 class _ContactListState extends State<ContactList> { 
     //create some fake data and assign to contactList 
-    List<Contact> contactList = MockContact.mocks(); 
+    List<Contact> contactList = MockContact.mockList(); 
```

Comment:
function name should be `mocks` which will return a List<> and `mock` which will return a single object.